src/components/login/loginForm/LoginForm.tsx

```diff
@@ -120,7 +123,7 @@ export const LoginForm = (): JSX.Element => {
                 error={
                   emailError ? 'Customer account with the given credentials not found.' : fieldState.error?.message
                 }
-                value={field.value}
+                value={customer ? customer.email : field.value}
```

Comment:
I'm not sure we have to use customer.email
It is better 
`value={field.value}`

Reply:
fixed